### PR TITLE
Add printing assignment

### PR DESCRIPTION
**Pull request by hroncok**
Several notes:

 * I consider creating extra repo with assignment an overkill here.
 * I've added assignment link to the printing page, to make it in a way "backwards compatible"

### CODE REVIEW

File: cs/printing.md [hunk 1]
```diff
@@ -33,3 +33,41 @@ Testovací objekt
...
+
+Úkol
+----
+
+### 1. úkol (1 bod)
+
+Tiskárna, která se vám objevila na stole, zrovna přijela na kontrolu.
+Je tedy potřeba abyste ověřili, zda dělá to, co by dělat měla.
+K tiskárně jste dostali tiskový profil, který na ní byl doteď používán.
+Předpokládejme, že není potřebo ho upravovat po kalibrační stránce
```
Comment (syky27) [inline cs/printing.md]: typo

**Review by syky27 — DISMISSED** [review]

File: cs/printing.md (hunk 1 above)
Comment (hroncok) [reply cs/printing.md]: changed. please squash if ok

**Comment by hroncok** [comment]
Rebased.

**Review by Hanyman8 — APPROVED** [review]
Looks OK

**Pull request merged**